help.html

```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: help
+permalink: /help/
+---
+
+<script>
+  window.location.replace("/user-guide");
+</script>
+
+<a href="/user-guide">This page has moved</a>
```

Comment:
```suggestion
<a href="/user-guide/">This page has moved</a>
```
Oh and btw. this fixes an actual issue caught by the link checker. `/user-guide` redirects to `/user-guide/` which is a completely uneccessary request to be made from a client perspective

Reply:
:+1: